frontend/js/GUI/Window.js

```diff
@@ -0,0 +1,341 @@
+// Configuration for imgui-js.
+SystemJS.config({
+  map: {
+    "imgui-js": "/js/GUI/imgui-js"
+  },
+  packages: {
+    "imgui-js": { main: "imgui.js" }
+  }
+});
+
+// Global vars.
+let ImGui;
+let ImGui_Impl;
+
+
+Promise.resolve().then(() => {
+    return System.import("imgui-js").then((module) => {
+        ImGui = module;
+        return ImGui.default();
+    });
+}).then(() => {
+    return System.import("imgui-js/example/imgui_impl").then((module) => {
+        ImGui_Impl = module;
+    });
+}).then(() => {
+    const canvas = document.getElementById("output");
+
+
+    ImGui.CreateContext();
+    ImGui_Impl.Init(canvas);
+
+    ImGui.StyleColorsDark();
+    //ImGui.StyleColorsClassic();
+
+
+    let done = false;
+    window.requestAnimationFrame(ImGuiLoop);
+
+    /**
+     * Function for making menubar object.
```

Comment:
Could we change "object" to window or something that shows it being a imgui menuebar

Reply:
Same with other windows.